facility-all-programs.ipynb

```diff
@@ -202,12 +201,42 @@
     "            i += 1\n",
     "        if ( i > 200 ):    # The map won't display with too many markers.\n",
     "            break\n",
-    "\n",
+    "            \n",
+    "    bounds = m.get_bounds()\n",
+    "    m.fit_bounds(bounds)\n",
+    "    \n",
     "    # Show the map\n",
     "    return m\n",
     "\n",
-    "map_of_facilities_in_cd = mapper(echo_data)\n",
-    "map_of_facilities_in_cd"
+    "map_of_facilities_in_zip = mapper(echo_data)\n",
+    "map_of_facilities_in_zip"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Graph data across all three programs, for all the facilities in the zip code. \n",
+    "We will only show those facilities that have - for any EPA program - spent at least 1 quarter out of the past 12 in non-compliance, or that have had 1 inspection over the past 5 years, or that had 1 enforcement action taken against them."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "chart_title = \"All Facilities - Past 3 Years of Violations and 5 Years of Inspections and Enforcement Actions\"\n",
```

Comment:
Getting a warning error from this code block:
```
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead
```

Reply:
I will try to fix that, though the block still resolves...

Reply:
fixed